src/components/block/FooterCTACard/FooterCTACard.js

```diff
@@ -0,0 +1,28 @@
+import PropTypes from "prop-types";
+import React from "react";
+import Button from "../../inline/Button";
+import s from "./FooterCTACard.module.css";
+
+const FooterCTACard = ({ subtitle, body, buttonText }) => (
+  <div className={s.container}>
+    <div className={s.subtitle}>{subtitle}</div>
+    <div className={s.body}>{body}</div>
+    <div className={s.button}>
+      <Button>{buttonText}</Button>
+    </div>
+  </div>
+);
+
+FooterCTACard.propTypes = {
+  subtitle: PropTypes.string,
+  body: PropTypes.string,
+  buttonText: PropTypes.string,
+};
+
+FooterCTACard.defaultProps = {
+  subtitle: PropTypes.string.isRequired,
+  body: PropTypes.string.isRequired,
+  buttonText: PropTypes.string.isRequired,
+};
```

Comment:
This is not a correct usage of `defaultProps`. `defaultProps` is meant to hold the value that will be used in place of a prop if it is not passed into the component. See https://reactjs.org/docs/typechecking-with-proptypes.html#default-prop-values for more details.

The `.isRequired` goes on the prop type definitions under `.propTypes`.